SDVModTest/UIElements/LocationOfTownsfolk.cs

```diff
@@ -244,26 +254,26 @@ private void DrawNPCLocationsOnMap(object sender, EventArgs e)
                                 int y = yBase + location.Value;
 
                                 Color color = character.CurrentDialogue.Count <= 0 ?
-                                    Color.Gray : Color.White;
+                                        Color.Gray : Color.White;
                                 ClickableTextureComponent textureComponent =
-                                    new ClickableTextureComponent(
-                                        character.name,
-                                        new Rectangle(x, y, 0, 0),
-                                        null,
-                                        character.name,
-                                        character.sprite.Texture,
-                                        headShot,
-                                        2.3f);
+                                        new ClickableTextureComponent(
+                                                character.name,
+                                                new Rectangle(x, y, 0, 0),
+                                                null,
+                                                character.name,
+                                                character.sprite.Texture,
+                                                headShot,
+                                                2.3f);
                                 Game1.spriteBatch.Draw(
-                                    character.sprite.Texture,
-                                    new Vector2(x, y),
-                                    new Rectangle?(headShot),
-                                    color,
-                                    0.0f,
-                                    Vector2.Zero,
-                                    2f,
-                                    SpriteEffects.None,
-                                    1f);
+                                        character.sprite.Texture,

```

Comment:
It would really help if you got rid of all the tab changes. It's hard for me to focus on the changes to integrate with your framework when you're throwing in all these formatting changes with them.